src/pathwayvis/components/map/views/map.component.html

```diff
@@ -10,7 +10,4 @@
   <div class="spinner" ng-if="ctrl.shared.isLoading()">
     <md-progress-circular id="map-progress" class="md-accent" md-diameter="85vh" layout-align="center center"></md-progress-circular>
   </div>
-  <div class="map-container" layout-fill layout="column" layout-align="center center">
```

Comment:
You deleted the whole div, that holds the Escher map. I already made this change locally, so I can roll this out with the leaky CSS fixes if you don't mind.

Reply:
Sounds good, sorry.